src/lib/_common.scss

```diff
@@ -117,34 +122,46 @@ $header-height: 48px;
     }
 }
 
-.bp {
-    align-items: center;
+.bp-body {
     background-color: $ffive;
-    border: 0 none;
     bottom: 0;
     display: flex;
-    flex-direction: column;
-    justify-content: center;
+    flex-direction: row;
     left: 0;
-    margin: 0;
-    outline: none;
-    -webkit-overflow-scrolling: touch;
-    padding: 0;
     position: absolute;
     right: 0;
     top: 0;
 
-    // Pseudo fullscreen for iOS Safari which doesn't support the fullscreen API
-    &.bp-fullscreen-unsupported {
-        bottom: 0;
-        left: 0;
-        position: fixed;
-        right: 0;
-        top: 0;
+    .bp-thumbnails-container {
```

Comment:
Are thumbnails specific to documents? If so, should this be in _docBase.scss? Or better yet in the thumbnails scss file

Reply:
There's a small chance we want to expand this to Tif files one day, but that's unlikely

Reply:
(related to my comment below)

Reply:
moved to _docBase.scss